src/routes/router.ts

```diff
@@ -1,8 +1,8 @@
 import { Application } from "express";
-import exampleRouter from "./example";
+import images from "./images";
 export default function router(app: Application): void {
   /**
    * Every source are specifed here
    */
-  app.use("/", exampleRouter);
+  app.use("/image",images);
```

Comment:
Use main path

Reply:
I didnt understand good, u want me to use router.ts and remove image.ts?

Reply:
instead `/image` will be `/`

Reply:
Ya en el ultimo commit estaria arreglado esta seccion

Reply:
Sorry, forget I was suposed to answer in english, my bad